src/configureStore.tsx

```diff
@@ -1,5 +1,13 @@
-import { Dispatch, Action, AnyAction } from 'redux';
-import { Store, applyMiddleware, combineReducers, createStore } from 'redux';
+import {
+  Action,
+  AnyAction,
+  Dispatch,
+  Reducer,
```

Comment:
I can't decide whether I think it's nicer, or more confusing, that we now import types the same as code. 🤷‍♂️

Reply:
> I can't decide whether I think it's nicer, or more confusing, that we now import types the same as code. 🤷‍♂️

I think it's nice. It illustrates how TypeScript is a first class language, not something bolted-on to the JS.

Reply:
> I think it's nice. It illustrates how TypeScript is a first class language, not something bolted-on to the JS.

💯